src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -6,19 +6,25 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMax.IdleMode;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.I2C.Port;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.awt.Color;
```

Comment:
Probably should use [`edu.wpi.first.wpilibj.util.Color`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/util/Color.html) instead of [`java.awt.Color`](https://docs.oracle.com/en/java/javase/11/docs/api/java.desktop/java/awt/Color.html), though in our use case (storing RGB values) they work identically.

Reply:
I was trying to use the wpilib one, but I couldn't find methods returning the individual r g and b values, so I resorted to this one since it worked

Reply:
Hm, looks like [`Color8Bit`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/util/Color8Bit.html) would be usable (also it can be passed into `AddressableLEDBuffer` if you feel like using it there too), but it should work either way.

Reply:
For the wpilib one you could use the `red`, `green`, and `blue` fields to get the individual values (it's pretty far down on the documentation page) but as Joseph mentioned, it's not too important.

Reply:
Can be fixed in a future pr, we'll circle back 😉😏